rubygem-kafo.spec

```diff
@@ -13,13 +13,15 @@ URL: https://github.com/theforeman/kafo
 Source0: http://rubygems.org/downloads/%{gem_name}-%{version}.gem
 %if "%{?scl}" == "ruby193" || (0%{?rhel} == 6 && 0%{!?scl:1})
 Requires: %{?scl_prefix}ruby(abi)
+Requires: %{?scl_prefix}rubygem(logging) < 2.0.0
```

Comment:
I think you'd simply want the same version numbers in both parts of the conditional, as this part isn't used exclusively say for Ruby 1.8.  The 1.8 compatibility ought to be handled by whoever packages rubygem(logging).